pkg/pipelines/initialise_test.go

```diff
@@ -0,0 +1,79 @@
+package pipelines
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestWriteResources(t *testing.T) {
+	tmpDir, cleanUp := makeTempDir(t)
+	defer cleanUp()
+	resources := map[string][]interface{}{
+		"01_roles/serviceaccount.yaml": []interface{}{fakeYamlDoc(1)},
+		"02_tasks/buildah_task.yaml":   []interface{}{fakeYamlDoc(1), fakeYamlDoc(2)},
+	}
+
+	_, err := writeResources(tmpDir, "", resources)
```

Comment:
There's no test for the second parameter here.

But see above, I don't think we even need the second parameter here.